s3/README.md

```diff
@@ -4,44 +4,75 @@ Module to create s3 bucket
+ | ---------------------------- | ------------- | ------------------- | ----------------- | -----------------  |
+ | region                       | string        |                     |                   |                    |
+ | profile                      | string        |                     |                   |                    |
+ | bucket_name                  | string        |                     |                   |                    |
+ | lazy_api_host                | string        | "https://lazy.apps.ops-drivevariant.com"  |                    | auto set at octopus|
+ | lazy_api_key                 | string        |                     |                   |auto set at octopus |
+ | role_arn                     | string        |      ""             |                   |                    |
+ | user_tags                    | object        |                     |user_tags = {team = "devops4" purpose = "s3-test3" owner = "naveen3"}| For `user_tags`, refer <https://github.com/variant-inc/lazy-terraform/tree/master/submodules/tags>   |
+ | octopus_tags                 | object        |                     | octopus_tags = { project = "actions-test3" space = "Default3"}| auto set at octopus|
+ | replication                  | string        |  false              |                   |                    |
+
+Tags module expect to have project and space in octopus tags but this is not mandatory if it is being run from octopus, they will get auto set.
+
+variable "octopus_tags" {
+  description = "Octopus Tags"
+  type = map(string)
+}
 
 ## Example .tfvars
 
 ```bash
 profile = "devops"
 region = "us-west-2"
-bucket_name = "naveen-ops-1"
-purpose = "devops"
-team = "devops"
-owner = "devops"
+bucket_name = "navin-ops-11"
 lazy_api_host = "https://lazy.apps.ops-drivevariant.com"
-lazy_api_key = "add_lazy_api_key_here"
-bucket = "lazy-tf-state20210107203535113800000001"
-key = "s3/default"
-dynamodb_table = "lazy_tf_state"
-s3_backend_region = "us-west-2"
-octopus_project_space = "test-space"
-octopus_project_name = "test-project"
-role_arn = "arn:aws:iam::123456789013:role/role_name"
+lazy_api_key = "####################"
+user_tags = {
+team = "devops4"
+purpose = "s3-test3"
+owner = "naveen3"
+}
+octopus_tags = {
+project = "actions-test3"
+space   = "Default3"
+}
+replication=true
+```
+
+## Example .tf file module reference
+
+```bash
+variable "octopus_tags" {
+  description = "Octopus Tags"
+  type = map(string)
+}
+variable "lazy_api_host" {
+  type = string
+}
+
+variable "lazy_api_key" {
+  type = string
+  sensitive = true
+}
+
+
+module "test_s3_module" {
+    source = "git::https://github.com/variant-inc/lazy-terraform.git//s3?ref=v1"
+        profile = "devops"
+        region = "us-west-2"
+        bucket_name = "navin-ops-39"
+        lazy_api_key = var.lazy_api_key # If run from octopus, this will be auto set
+        lazy_api_host = var.lazy_api_host # If run from octopus, this will be auto set
+        user_tags = {
```

Comment:
Please format properly with proper indentation. Check the entire file for similar issues.

Reply:
Corrected, not sure if you are looking at any specific lines.